components/picker/metadata/picker.yml

```
@@ -13,6 +13,7 @@ sections:
       - The mod custom property `--mod-line-height-100` has been renamed to `--mod-button-line-height`.
       - The mod custom property `--mod-sans-font-family-stack` has been renamed to `--mod-button-font-family`.
       - The mod custom property `--mod-animation-duration-100` has been renamed to `--mod-button-animation-duration`.
+      - The deprecated mod custom property `--mod-picker-spacing-text-to-alert-icon-inline-start` was removed. This was replaced with `--mod-picker-spacing-text-to-alert-icon-inline-start`.
```

These two mods are the same, should this be:

```suggestion
      - The deprecated mod custom property `--mod-picker-spacing-text-to-alert-icon-inline-start` was removed. This was replaced with `--spectrum-picker-spacing-text-to-icon-inline-end`.
```